bin/branding.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+import os
+import string
+import fnmatch
+import sys
+from optparse import OptionParser
+
+path = os.path.dirname(os.path.realpath(__file__))
+branding_values = dict()
+
+# Read branding environment variable list from branding file
+# For each variable read the corresponding env variable and store to branding map
+with open(os.path.join(path, 'branding.list'), 'r') as branding:
+	lines = filter(lambda x: x and not x.startswith('--'), branding.read().splitlines())
+	branding_values = {x:os.environ.get(x) for x in lines}
+
+# Read list of environment variables from file, where each is a path to a file
```

Comment:
Chicken and egg problem...
You don't know where all the files are until you parse them (make's include).  All I want to know is to point at the first file that may recursively include others..